eureka-common/src/main/java/edu/emory/cci/aiw/cvrg/eureka/common/comm/clients/ServicesClient.java

```diff
@@ -329,73 +329,57 @@ public SystemElement getSystemElement(String inKey, boolean summarize) throws Cl
 			return result.get(0);
 		}
 	}
-
-	public List<TimeUnit> getTimeUnits() throws ClientException {
-		final String path = "/api/protected/timeunit/list";
-		return doGet(path, TimeUnitList);
-	}
-
+        
+        
 	public List<TimeUnit> getTimeUnitsAsc() throws ClientException {
-		final String path = "/api/protected/timeunit/listasc";
+               final String path = "/api/protected/timeunits";
```

Comment:
I see some minor tab and whitespace inconsistencies. Eureka and all of its subprojects use tabs for indendation. All other projects use spaces for indentation.